lib/web_ui/dev/steps/run_suite_step.dart

```diff
@@ -192,17 +194,26 @@ class RunSuiteStep implements PipelineStep {
       workDirectory.deleteSync(recursive: true);
     }
     final bool isWasm = suite.testBundle.compileConfigs.first.compiler == Compiler.dart2wasm;
+    final bool singleThreaded = suite.runConfig.forceSingleThreadedSkwasm || !suite.runConfig.crossOriginIsolated;
+    final String rendererName = switch (renderer) {
+      Renderer.skwasm => singleThreaded ? 'skwasm_st' : 'skwasm',
+      _ => renderer.name,
+    };
+
+    final dimensions = <String, String> {
+      'Browser': suite.runConfig.browser.name,
+      if (isWasm) 'Wasm': 'true',
+      'Renderer': rendererName,
+      if (variant != null) 'CanvasKitVariant': variant.name,
+    };
+    print('Created Skia Gold Client. dimensions: $dimensions');
     final SkiaGoldClient skiaClient = SkiaGoldClient(
       workDirectory,
-      dimensions: <String, String> {
-        'Browser': suite.runConfig.browser.name,
-        if (isWasm) 'Wasm': 'true',
-        'Renderer': renderer.name,
-        if (variant != null) 'CanvasKitVariant': variant.name,
-      },
+      dimensions: dimensions,
     );
 
     if (await _checkSkiaClient(skiaClient)) {
+      print('Successfully checked Skia Gold Client');
```

Comment:
Are these leftover debug prints or are they intended to be part of the output (e.g. to help debug flaky tests on CI)

Reply:
I found them actually kind of useful enough to leave in. They only are printed once for each suite when they run, so they aren't particularly chatty. If you don't like them in here I can remove them, but it's actually nice to be able to make sure we created a skia gold client with the right dimensions.

Reply:
Seems fine to keep in, then.